projects/canopy/src/lib/forms/checkbox-group/checkbox-group.notes.ts

```diff
@@ -0,0 +1,81 @@
+export const notes = `
+# Checkbox Group Components
+
+## Purpose
+Provides a set of components to implement Checkbox groups in a form. The Checkbox Group component is a container which displays the label with an optional hint and should contain two or more Checkbox components. The Checkbox component represents a single checkbox. The Hint component may be used to provide extra context to the user. The Checkbox component is a variant of the Toggle component.
+
+## Usage
+Import the CheckboxGroup Module into your application:
+
+~~~js
+@NgModule({
+  ...
+  imports: [LgCheckboxGroupModule],
+})
+~~~
+
+and in your HTML:
+
+~~~html
+<lg-checkbox-group [inline]="true" formControlName="color">
```

Comment:
This super minor, but because it's a checkbox group we should pluralise this example.
`formControlName="colors"`

`<lg-hint>Please select all colours that apply</lg-hint>` (or something to that effect)

Reply:
Done.